mmdet/utils/ascend_util.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+
+def set_index(ori_tensor, mask, new_value, neg=False):
+    if mask is None:
+        return ori_tensor
+    else:
+        if neg:
+            return ori_tensor * mask + new_value * (1 - mask)
+        else:
+            return ori_tensor * (1 - mask) + new_value * mask
+
+
+def images_to_levels(target, num_levels):
```

Comment:
There is already an `images_to_levels` in this repo, it might be better to call it another name like [xxx]_images_to_levels